<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1e1e1e;</string>
  </property>
  <widget class="QFrame" name="topBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>57</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnFC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cbSerial">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>14</y>
      <width>76</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>8</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWebEngineView" name="mapView" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>62</y>
     <width>1220</width>
     <height>705</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
